### PR TITLE
Mailchimp signup

### PR DESCRIPTION
**Pull request by ChefAndy**
Killed our internal mailing list sign-up functionality and pointed our mailing list sign-up form right at our mailchimp campaign.

### CODE REVIEW

File: capstone/capapi/migrations/0016_delete_mailinglist.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Generated by Django 2.2.4 on 2019-08-23 18:38
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```
Comment (jcushman) [inline capstone/capapi/migrations/0016_delete_mailinglist.py]: Let's wait to delete the model until the rest of this lands, so we don't have to be too precise about the timing of copying the data over.

File: capstone/capweb/templates/index.html [hunk 2]
```diff
@@ -609,10 +614,15 @@ <h2 class="section-title pt-0 mb-3">
         </div>
         <div class="d-none d-md-block col-5 col-md-5 col-lg-3 offset-1 offset-sm-0">
 
-          <form class="email_form_large" action="{% url 'subscribe' %}" method="post">
-                {% bootstrap_form subscribe_form %}
-                <div class="add">Add your email to keep in touch</div>
-                <button class="btn-secondary subscribe" type="submit">Subscribe!</button>
+          <form action="https://law.us3.list-manage.com/subscribe/post?u=4290964398813d739f2398db0&amp;id=e097736c6f"
```
Comment (jcushman) [inline capstone/capweb/templates/index.html]: Can we make the magic numbers here (and same elsewhere) into settings? That way dev/staging sites won't post to the real mailing list.

**Review by jcushman — CHANGES_REQUESTED** [review]
Thanks! Just a couple of tweaks.

File: capstone/capweb/templates/index.html (hunk 2 above)
Comment (ChefAndy) [reply capstone/capweb/templates/index.html]: Sure.

File: capstone/capapi/migrations/0016_delete_mailinglist.py (hunk 1 above)
Comment (ChefAndy) [reply capstone/capapi/migrations/0016_delete_mailinglist.py]: Ok.

**Comment by jcushman** [comment]
Thanks!

**Pull request merged**